<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QWidget" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="rightMargin">
      <number>36</number>
     </property>
     <item>
      <widget class="QTableView" name="stars_table_view"/>
     </item>
     <item>
      <widget class="QCheckBox" name="filter_check_box">
       <property name="text">
        <string>See my stars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view_push_button">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>26</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="add_push_button">
       <property name="text">
        <string>Add star</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_line_edit">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ra_line_edit">
       <property name="placeholderText">
        <string>Ra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Dec_line_edit">
       <property name="placeholderText">
        <string>Dec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="diameter_line_edit">
       <property name="placeholderText">
        <string>Diameter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="search_line_edit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="searched_list_widget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
